userspace/libscap/scap.h

```diff
@@ -144,6 +145,31 @@ typedef struct scap_stats
 	uint64_t n_tids_suppressed; ///< Number of threads currently being suppressed.
 }scap_stats;
 
+
+typedef union scap_stats_v2_union {
```

Comment:
```suggestion
typedef union scap_stats_v2_value {
```
Just a nit naming suggestion, disregard if you don't like it.